### PR TITLE
Move accounts* from state to environment

### PR DESCRIPTION
**Pull request by pattyshack**

### CODE REVIEW

File: fvm/state/state_test.go [hunk 1]
```diff
@@ -12,6 +12,14 @@ import (
 	"github.com/onflow/flow-go/fvm/utils"
 )
 
+func createByteArray(size int) []byte {
```
Comment (Tonix517) [inline fvm/state/state_test.go:15]: what is this function for?

**Review by Tonix517 — APPROVED** [review]

File: fvm/state/state_test.go (hunk 1 above)
Comment (pattyshack) [reply fvm/state/state_test.go:15]: it's copied from accounts_test.go.  we need to duplicated it here since it's also used by these tests

**Review by SaveTheRbtz — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by ramtinms** [comment]
I would recommend adding more details to the PRs, specially in this PR its not clear to me why we are moving accounts to environment, previously we defined every stateful provider to be inside the state package, but this is kinda changing that

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**